### PR TITLE
Update formatCurrency helper to match Ember-CLI

### PR DESCRIPTION
**Pull request by jsmestad**
When running `ember g helper format-currency` the generated file follows a different format than the guides show. This is using `ember-cli 2.2.0-beta.3`

### CODE REVIEW

File: source/templates/writing-helpers.md [hunk 1]
```diff
@@ -42,7 +42,9 @@ That file should export a function wrapped with [`Ember.Helper.helper()`][1]:
 [1]: http://emberjs.com/api/classes/Ember.Helper.html#method_helper
 
 ```app/helpers/format-currency.js
-export default Ember.Helper.helper(function(params) {
+import Ember from 'ember';
```
Comment (locks) [inline source/templates/writing-helpers.md]: can you omit the import please?

**Comment by acorncom** [comment]
@jsmestad Background on the request above: what you're putting in is valid, but we're not quite ready to start putting in import statements across all the docs due to some future changes that may affect that.  Thanks!

**Comment by jsmestad** [comment]
@acorncom it may be useful in that case to just call out that running that command could have a different looking output. I'm happy to draft that if your ok with accepting something like that.

**Comment by locks** [comment]
We're already omitting `import`s everywhere except the tutorial. Another thorn is that the blueprints are in CLI until Ember gets completely addonified.

tl;dr no call out

**Comment by jsmestad** [comment]
alright better luck next time ;)

**Comment by locks** [comment]
what about the other changes? :x

**Comment by jsmestad** [comment]
@locks doh! your only talking about the import line, not the export. &#^!!

**Comment by jsmestad** [comment]
Removed `import` statement as requested. Rest should be :+1:

**Comment by acorncom** [comment]
@jsmestad Sweet.  Mind squashing your commits?  We'll merge it right after that :+1:

**Comment by jsmestad** [comment]
@locks @acorncom squashed.

**Comment by locks** [comment]
@homu r+
